### DISCUSSION TITLE
how many connections can msquic support for server， server is in CDN

### DISCUSSION THREAD

**Discussion by ycchildcoder** [microsoft/msquic#4004] [General]
1、how many connections can msquic support for server， server is in CDN

2、How to set up a single connection faster

**Comment by nibanks** [comment]
I don't have a good answer for you, since it simply depends on your machine and configuration. MsQuic is built for server scale and performance, so it should be able to handle LOTS of connection, but you need to try it out yourself to see what your machine can handle. If you find anything unexpected, feel free to let us know and we can help investigate.
